client/state/login/actions.js

```diff
@@ -434,7 +434,10 @@ export const logoutUser = ( redirectTo ) => ( dispatch, getState ) => {
 
 export const fetchOAuth2ClientData = ( clientId ) => dispatch => {
 	dispatch( {
-		type: OAUTH2_CLIENT_DATA_REQUEST
+		type: OAUTH2_CLIENT_DATA_REQUEST,
```

Comment:
> Just wondering, why is this a thunk and not just: [...]

@yurynix I was wondering the same thing until I noticed the `wpcom` call below :slightly_smiling_face: